rotkehlchen/tests/conftest.py

```diff
@@ -246,13 +250,26 @@ def fixture_vcr_base_dir() -> Path:
     log.debug(f'At VCR setup, {current_branch=} {root_dir=}')
 
     checkout_proc = Popen(f'cd "{root_dir}" && git fetch origin && git checkout {current_branch}', shell=True, stdout=PIPE, stderr=PIPE)  # noqa: E501
-    _, stderr = checkout_proc.communicate(timeout=SUBPROCESS_TIMEOUT)
+
+    try_fallback_branch = False
```

Comment:
Why this logic with `try_fallback_branch` and not just add one more `stderr` check? It only times out? Why? When does it happen?

Reply:
In the case of the `test` branch that we use to trigger the same work as in the nightlies I found that the command was timing out instead of failing directly. This case is the case where the `current_branch` doesn't exists. I tried to reproduce it locally and works fine so I believe is some special case in the CI

Reply:
This sounds wrong. You should try to figure out why it is timing out.